client/src/components/addSourceWizard/addSourceWizard.js

```diff
@@ -226,4 +229,7 @@ const mapStateToProps = function(state) {
   return { ...state.addSourceWizard.view };
 };
 
-export default connect(mapStateToProps, mapDispatchToProps)(AddSourceWizard);
+export default connect(
+  mapStateToProps,
+  mapDispatchToProps
+)(AddSourceWizard);
```

Comment:
Is this due to line length lint errors? Doesn't seem to be too long.

Reply:
Yeah, it looks like it's an aspect of `prettier` having to do with function calls... I think this is related:
- https://github.com/prettier/prettier/issues/1420